content/source/docs/enterprise/sentinel/import/tfplan.html.md

```diff
@@ -415,6 +407,15 @@ import "tfplan"
 main = rule { tfplan.resources.null_resource.foo[0].applied.triggers.foo is "bar" }
 ```
 
+-> Note that some values will not be available in the `applied` state because
+they cannot be known until the plan is actually applied. In Terraform 0.11 or
+earlier, these values are represented by a placeholder (the UUID value
+`74D93920-ED26-11E3-AC10-0800200C9A66`) and in Terraform 0.12 or later they
+are simply omitted (a null value). **In either case**, you should instead use the
```

Comment:
Just ahead of a larger review - remember that it's not a _null_ value, but an _undefined_ value.

Null itself is an explicit value in Sentinel (see [the spec](https://docs.hashicorp.com/sentinel/language/spec#null)).

Reply:
Gotcha, fixed!